exporter/tanzuobservabilityexporter/metrics_test.go

```diff
@@ -197,6 +196,270 @@ func TestGaugeConsumerMissingValue(t *testing.T) {
 	assert.Len(t, allLogs, expectedMissingValueCount)
 }
 
+// Tests that the histogramConsumer correctly delegates to its
+// histogramDataPointConsumers. This tests delta histograms
+func TestHistogramConsumerDeltaAggregation(t *testing.T) {
+	// TODO: Change this test when delta histograms are supported.
+	countAttributeForEachDataPoint := []uint64{2, 5, 10}
+	deltaMetric := newHistogramMetricWithDataPoints(
+		"delta.metric",
+		pdata.MetricAggregationTemporalityDelta,
+		countAttributeForEachDataPoint)
+	sender := &mockGaugeSender{}
+	cumulativeConsumer := &mockHistogramDataPointConsumer{}
+	consumer := newHistogramConsumer(
+		cumulativeConsumer, nil, sender, nil)
+	var errs []error
+	consumer.Consume(deltaMetric, &errs)
+	assert.Len(t, errs, 1)
+	consumer.PushInternalMetrics(&errs)
+	assert.Len(t, errs, 1)
+}
+
+// Tests that the histogramConsumer correctly delegates to its
+// histogramDataPointConsumers. This tests cumulative histograms
+func TestHistogramConsumerCumulativeAggregation(t *testing.T) {
+	countAttributeForEachDataPoint := []uint64{2, 5, 10}
+	cumulativeMetric := newHistogramMetricWithDataPoints(
+		"cum.metric",
```

Comment:
Can we spell out "cumulative.metric" here?

Reply:
Done.